Destiny.ScrimTracker.Api/Controllers/MatchController.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using Destiny.ScrimTracker.Api.Requests;
+using Destiny.ScrimTracker.Logic.Models;
+using Destiny.ScrimTracker.Logic.Repositories;
+using Destiny.ScrimTracker.Logic.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Destiny.ScrimTracker.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class MatchController : Controller
+    {
+        private readonly IMatchService _matchService;
+        private readonly IGuardianService _guardianService;
+
+        public MatchController(IMatchService matchService, IGuardianService guardianService)
+        {
+            _matchService = matchService;
+            _guardianService = guardianService;
+        }
+        
+        // GET
+        public IActionResult Index()
+        {
+            var matchResults = _matchService.GetMatchResults();
+            return View(matchResults);
+        }
+
+        [Authorize]
+        [Route("match")]
```

Comment:
Can we keep this RESTful? 
```suggestion
        [Route("new")]
```

Reply:
This is fixed in the next PR